tests/ci_build/Dockerfile.ubuntu1404_crosstool

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+MAINTAINER Aran Khanna <arankhan@amazon.com>
+
+# UPDATE BOX
```

Comment:
please consider to use scripts under installs. so that the we can reuse docker caches. check Dockerfile.cpu